### ISSUE TITLE
Webhook: add event information

### ISSUE THREAD

**Issue by GrinchUA** [bluecherrydvr/bluecherry-apps#411]
https://github.com/bluecherrydvr/bluecherry-apps/blob/d55f8e3002f53b0163d445294c5549f8355609b4/www/lib/mailer.php#L102

The function contains information about the event, why not send it to the webhook, this information will be useful for processing.

**Comment by curtishall** [comment]
I agree it would be useful.  If you want something sooner we accept PRs, otherwise I'll keep it in mind for after the 3.1 stable release.

**Comment by kala13x** [comment]
Hi, testing has shown that mailer.php already sends detailed information about the event to the webhook, this is the example of the raw content  of mentioned information:
```
{
  "event_name": "motion_event",
  "event_datetime": "2021-04-17T02:08:59+04:00",
  "device_id": "2",
  "device_name": "TestCam",
  "dvr_name": "Bluecherry DVR v3"
} 
```
